phpunit.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Certainly. If this were a framework or some other project meant for others to use as a base to start their own projects, then I can understand leaving phpunit.xml.dist, but this is not such a project. The .dist version is meant to be used as a template for you to quickly make your own setting file, but not necessarily use it as a default. It's misleading to a developer in my opinion. To me it says "you can run tests if you want, but we don't care enough to define our own test setup, so here's a file to get you started."